KUP.Authentication.Api/Controllers/AuthenticationController.cs

```diff
@@ -14,7 +14,10 @@ namespace KUP.Authentication.Api.Controllers
     [Route("api/[controller]")]
     public class AuthenticationController : Controller
     {
-
+        // this is very bad code.. 
+        int myInt = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make 'myInt' 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.k8s.int.kaplan.edu/coding_rules#rule_key=csharpsquid%3AS2933)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'myInt'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.k8s.int.kaplan.edu/coding_rules#rule_key=csharpsquid%3AS1144)